src/components/SearchGridInfiniteLoad.vue

```diff
@@ -133,7 +133,7 @@ export default {
     searchChanged() {
       this.showGrid = false;
       this.$store.commit(SET_IMAGES, { images: [] });
-      this.currentPage = 0;
+      this.currentPage = 1;
```

Comment:
Can you keep the changes limited to this one?

Reply:
Sure I can.